### ISSUE TITLE
`require-param` not working on angular component class methods

### ISSUE THREAD

**Issue by Taragryen** [gajus/eslint-plugin-jsdoc#1166]
## Expected behavior
Report an error: Missing JSDoc @param "btnState" declaration.

## Actual behavior
No error reported

## ESLint Config

```js
"jsdoc/require-param": [
  "error",
  {
    "contexts": [
      "FunctionDeclaration",
      "TSFunctionType",
    ]
  }
], //not working on class methods
```

## ESLint sample

```ts
this is an angular component class method
  /**
   * openConfirmModal
   * @memberof CreateEditTestWizardComponent
   */
  public openConfirmModal(btnState: string) {
    this.modalBtnState = btnState;
    this.openModal();
  }
```

## Environment

- Node version: v20.6.1
- ESLint version: v8.42.0
- `eslint-plugin-jsdoc` version: latest

**Comment by Taragryen** [comment]
<img width="647" alt="image" src="https://github.com/gajus/eslint-plugin-jsdoc/assets/44627712/2cca9996-31c4-47b5-9610-d9b5a5740ddc">
I followed the `require-param` example to configure it

**Comment by brettz9** [comment]
Use `MethodDefinition` instead of `TSFunctionType`.

In that example, it works because `TSFunctionType` is finding the type annotation `(id: number) => string`, not the method `TestMethod`.

**Comment by brettz9** [comment]
Can you provide a minimal example where it does not work?

**Comment by Taragryen** [comment]
After reinstalling the eslint extension, require-param is back to work now
Sorry for taking up your time
